specifyweb/frontend/js_src/lib/components/FormEditor/Create.tsx

```diff
@@ -132,18 +132,18 @@ function ListViews({
   const { disciplines } = useOutletContext<FormEditorOutlet>();
 
   return (
-    <div className="flex flex-col gap-4">
+    <Ul className="flex flex-col gap-4">
       <h3 className={className.headerPrimary}>{header}</h3>
```

Comment:
this is not a correct markup:. headers can't be inside of `<ul>`

a correct markup may look like this:
```html
<ul>
  <li>
    <h2>Section Title</h2>
    <ul>
      <li>
        <h3>Subsection Title</h3>
        <ul>
          <li>
            <h4>Subsubsection Title</h4>
            <!-- Further content here -->
          </li>
        </ul>
      </li>
    </ul>
  </li>
</ul>
```

note, that you might also want to convert div on line 74 into a `<Ul>`, and then the element on line 135 into a `<li>`

Reply:
not fully resolved yet.

see the comment I left above:
> note, that you might also want to convert div on line 74 into a `<Ul>`, and then the element on line 135 into a `<li>`

in current implementation you have two `<ul>`, with only one `<li>` in each - why not just replace the div on line 74 and then you won't have to create two separate `<ul>` (simpler, and better semantics)

Reply:
sorry, the latest commit is quite wrong

1. you converted from `div` to `Ul` correctly. BUT:
2. changing all `Ul` to `li` is not correct. you can't have `li` inside of `li` without any `<ul>` in between. Instead of this incorrect change, you should have just removed the `Ul` in `ListViews` as we now have `Ul` in `CreateFormDefinition`, and then move over the class names `Ul` had over to the root `li` in `ListViews`

I made the necessary changes in 6e0b18b

So in summary, the structure of the DOM tree should be something like this:
```html
<ul>
  <li>
    <h3>Some header</h3>
    <ul>
      <li>
        <h4>Some subheader</h4>
         <ul>
          <li>
            <h5>Some sub-sub header</h5>
            ...
          </li>
          <li>
            ...
          </li>
        </ul>
      </li>
      <li>
        ...
      </li>
    </ul>
  </li>
  <li>
    ...
  </li>
  <!-- you can have as many li as you want -->
</ul>
```

(note that Ul is pretty much just ul so you can treat them as the same for purposes of this sample)

(you can also see a similar dom structure in this code sample: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#ordered_list_inside_unordered_list, except they don't wrap headers in `<h2>`, and use `ol` for nested list)

The code is now semantically correct. Can you double-check that the styling is still correct, and if not adjust the class names as necessary?

also, please tell if I should have explained things in some other way to reduce the confusion in the future